Tests/QuantConnect.Tests.csproj

```diff
@@ -290,6 +290,9 @@
     <None Update="TestData\spy_with_hilbert.csv">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="TestData\spy_tsf.csv">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Welcome to LEAN @krutik2966! Please attach this file with the commit so we can run the tests.

Where did you source the data from? It needs to be a reliable test dataset.

Reply:
Thank you so much @jaredbroad!

I've only used the example data provided at [https://tulipindicators.org/tsf](https://tulipindicators.org/tsf). So far, I've only applied the indicator to this specific dataset and haven't tested it on any other data sources. At first, I was extracting the data from this file, but in the final version of the code, I decided to directly put the data into the `Tests/Indicators/TimeSeriesForecastTests.cs` file.

Reply:
Thank you. Please revert and follow the pattern of other indicator tests by providing the spy_tsf test file